201-vnet-to-vnet-bgp/azuredeploy.json

```diff
@@ -0,0 +1,351 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location1": {
+            "type": "string",
+            "metadata": {
+                "description": "Location where first VNET, Gateway, PublicIP and Connection will be deployed."
+            }
+        },
+        "location2": {
+            "type": "string",
+            "metadata": {
+                "description": "Location where second VNET, Gateway, PublicIP and Connection will be deployed."
+            }
+        },
+        "virtualNetworkName1": {
```

Comment:
move to variable and use the location param as part of the name, e.g.
"vnet1Name": "[concat("vNet1-", replace(parameters('location1'), ' '. ''))]"